docs/source/deploypeer/peerchecklist.md

```diff
@@ -0,0 +1,406 @@
+
+## address
+```
+# When used as peer config, this represents the endpoint to other peers
+# in the same organization. For peers in other organization, see
+# gossip.externalEndpoint for more info.
+# When used as CLI config, this means the peer's endpoint to interact with
+address: 0.0.0.0:7051
+```
+- **`address`**: (Default value should be overridden.) Specify the address that other peers in the organization use to connect to this peer, for example, `peer0.org1.example.com:7051`.
+
+## mspConfigPath
+```
+mspConfigPath: msp
+```
+- **`mspConfigPath`**: (Default value should be overridden.) This is the path to the peer's local MSP, which must be created before the peer can be deployed. The path can be absolute or relative to `FABRIC_CFG_PATH` (by default, it is `/etc/hyperledger/fabric` in the peer image). Unless an absolute path is specified to a folder named something other than "msp", the peer defaults to looking for a folder called “msp” at the path (in other words, `FABRIC_CFG_PATH/msp`) and when using the peer image: `/etc/hyperledger/fabric/msp`. If you are using the recommended folder structure described in the [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topic, it would be relative to the FABRIC_CFG_PATH as follows:
+`config/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/msp`
+
+## localMspId
+```
+# Identifier of the local MSP
+# ----!!!!IMPORTANT!!!-!!!IMPORTANT!!!-!!!IMPORTANT!!!!----
+# Deployers need to change the value of the localMspId string.
+# In particular, the name of the local MSP ID of a peer needs
+# to match the name of one of the MSPs in each of the channel
+# that this peer is a member of. Otherwise this peer's messages
+# will not be identified as valid by other nodes.
+localMspId: SampleOrg
+```
+
+- **`localMspId`**: (Default value should be overridden.) This is the value of the MSP ID of the organization the peer belongs to. Because peers can only be joined to a channel if the organization the peer belongs to is a channel member, this MSP ID must match the name of at least one of the MSPs in each of the channels that this peer is a member of.
+
+## fileSystemPath
+```
+# Path on the file system where peer will store data (eg ledger). This
+# location must be access control protected to prevent unintended
+# modification that might corrupt the peer operations.
+fileSystemPath: /var/hyperledger/production
+```
+- **`fileSystemPath`**: (Default value should be overridden.) This is the path to the ledger and installed chaincodes on the local filesystem of the peer. It can be an absolute path or relative to FABRIC_CFG_PATH. It defaults to `/var/hyperledger/production`. The user running the peer needs to own and have write access to this directory.
+
+## gossip
+
+```
+gossip:
```

Comment:
I like how you pulled out the important gossip properties and list them all together.